### DISCUSSION TITLE
Support TCP on port 443

### DISCUSSION THREAD

**Discussion by EgidioCaprino** [haugene/docker-transmission-openvpn#1933] [Ideas]
## Before creating this request I have:
**REQUIRED**
- [x] Read through the pinned issues for related problems
- [x] Searched for similar [provider issues][pi] and [container issues][ci]
- [x] [Read the documentation][rd], especially the troubleshooting section and FAQ
- [x] Tried to add as much relevant information to the request as possible
- [x] Agreed that my issue will be closed if I do not follow this template and will remain closed until I complete the template

[pi]: https://github.com/haugene/vpn-configs-contrib/issues
[ci]: https://github.com/haugene/docker-transmission-openvpn/issues
[rd]: https://haugene.github.io/docker-transmission-openvpn/

### Is your feature request related to a problem? Please describe.
**REQUIRED**
```txt
Some ISP engages in bandwidth throttling. A workaround is to switch from UDP to TCP on port 443. More details here: https://mullvad.net/it/help/improve-slow-speeds-throttling/
```

### Describe the solution you'd like
**REQUIRED**
```txt
I couldn't find any way to change the OpenVPN protocol and port. It would be great to have OPENVPN_PROTOCOL and OPENVPN_PORT environment variables.
```

### Describe alternatives you've considered
**REQUIRED**
```txt
Creating a custom it_all.ovpn file and mounting it to /etc/openvpn/mullvad/it_all.ovpn
```
### Additional context
*Optional*
```txt
None
```

**Comment by pkishino** [comment]
all that is needed for this is to updated the vpn configs or as you did above, create a custom profile specifying the port.
Not really anything to be done on this container directly
